csharp/Adapter/Microsoft.Spark.CSharp/Proxy/Ipc/SparkContextIpcProxy.cs

```diff
@@ -20,7 +20,7 @@ internal class SparkContextIpcProxy : ISparkContextProxy
     {
         private JvmObjectReference jvmSparkContextReference;
         private JvmObjectReference jvmJavaContextReference;
-        private JvmObjectReference jvmAccumulatorReference;
+        internal JvmObjectReference jvmAccumulatorReference;
```

Comment:
instead of this I suggest creating an internal getter property like it is done for JvmSparkContextReference below